src/NUnitFramework/framework/Constraints/CollectionOrderedConstraint.cs

```diff
@@ -63,6 +65,10 @@ public CollectionOrderedConstraint Descending
             get
             {
                 descending = true;
+
+                if (this.properties != null && this.properties.Count != 0)
+                    this.properties[this.properties.Keys.Max()] = true;
+
```

Comment:
I suspect you have changed behavior of `Is.Ordered.Descending.By((SomeProp")` here. If so, we're all at fault because there's no test of it. Can you add one?